KBH/6Day/예산.js

```diff
@@ -0,0 +1,18 @@
+function solution(d, budget) {
+    const sorted = d.toSorted((a, b) => a - b);
+    
+    let sum = 0;
+
+    let count = 0;
```

Comment:
```suggestion
    const sorted = d.toSorted((a, b) => a - b);
    let sum = 0;
    let count = 0;
```
변수 선언문은 붙여서 쓰는게 더 보기 좋아보이네요.